.mailmap

```diff
@@ -185,6 +185,7 @@
 #
 # See also: 'git shortlog --help' and 'git check-mailmap --help'.
 #
+
```

Comment:
Why was the newline added? Was it generated by the script?

Reply:
Maybe my text editor added the new line- since I added my name at the bottom of the file.

While I have you here, I found another bug that I'd like to discuss.

Reply:
Please open a new issue or send a note to the mailing list.